en/docs/chapter_introduction/algorithms_are_everywhere.md

```diff
@@ -25,17 +25,17 @@ Before formally discussing algorithms, there's an interesting fact worth sharing
 === "<5>"
     ![Binary Search in Dictionary Step 5](algorithms_are_everywhere.assets/binary_search_dictionary_step5.png)
 
-This essential skill for elementary students, looking up a dictionary, is actually the famous "Binary Search" algorithm. From a data structure perspective, we can consider the dictionary as a sorted "array"; from an algorithmic perspective, the series of actions taken to look up a word in the dictionary can be viewed as "Binary Search."
+Looking up a dictionary, a must-have skill for primary school students, is actually the famous "binary search" algorithm. From the perspective of data structure, we can regard the dictionary as a sorted "array"; from the perspective of algorithm, we can regard the above series of dictionary lookup operations as "binary search".
 
 **Example 2: Organizing Playing Cards**. When playing cards, we need to arrange the cards in our hand in ascending order, as shown in the following process.
 
 1. Divide the playing cards into "ordered" and "unordered" sections, assuming initially the leftmost card is already in order.
 2. Take out a card from the unordered section and insert it into the correct position in the ordered section; after this, the leftmost two cards are in order.
-3. Continue to repeat step `2.` until all cards are in order.
+3. Repeat step `2`, inserting one playing card from the unsorted section into the sorted section each round, until all the cards are in order.
```

Comment:
I think `Repeat step `2.` until all the cards are in order`. is clearer, as it emphasizes the repetitive nature of the algorithm.

Reply:
Fixed. This expression is more concise!